cardano-cli/src/Cardano/CLI/EraBased/Options/Transaction.hs

```diff
@@ -30,7 +31,7 @@ pTransactionCmds
   => ShelleyBasedEra era
   -> EnvCli
   -> Maybe (Parser (TransactionCmds era))
-pTransactionCmds era envCli =
+pTransactionCmds era' envCli =
```

Comment:
Do you need this renaming if you have `hiding (era)` in the imports?

Reply:
I think I don't. I did this because hydra was still complaining after `hiding`.

Reply:
Also the field name `era` isn't the best one I guess. 🤔

Reply:
@carbolymer> Yeah I'd probably not use `era` as a field name indeed.

Reply:
Renamed